HdmiCec/HdmiCec.json

```diff
@@ -94,8 +109,53 @@
                 ]
             }
         },
+        "getDeviceList":{
+            "summary": "Gets the number of connected source devices and system information for each device. The information includes logicalAddress,OSD name and vendor ID.\n  \n### Event \n\n No Events",
```

Comment:
Gets the list of number of CEC enabled devices connected and system information for each device.

Reply:
Updated changes as per your review comments
Please refer changes in #2155